src/Tests/Moryx.Tests/Bindings/BindingResolverExtensionTests.cs

```diff
@@ -125,6 +125,18 @@ public void CallRemoveRemovesItemFromChain(int removeCount)
             Assert.AreEqual(5 - removeCount, CountChainLinks(_resolver));
         }
 
+        [TestCase(null, Description = "should return null value")]
+        [TestCase("This is a wrong binding", Description = "Should return original string value")]
```

Comment:
Your invalid binding string should reference a property on `data` that does not exist or is a typo of the property.

For example `"This text used {Branch.Nmae} on data which was not found"`.